started learning htlm and css for webpage

17chua opened the pull request:
Created a basic page explaining our project. It lacks functionality and looks, but I plan to learn javascript and get better at CSS next week.

17chua commented:
<img width="1343" alt="Screenshot 2022-11-11 at 3 02 20 PM" src="https://user-images.githubusercontent.com/77593979/201430302-a3301456-51b0-4be9-b822-afde1aba36a6.png">

zhangathan-jon reviewed (approved):
Webpage is well organized and well-written; this is a great start!

17chua commented:
(11/18/22)
Updated the main page to include a navigation bar (with working home / make a query pages/links) as well as a make a query button. Also moved CSS stuff to a separate file. 
<img width="1118" alt="Screenshot 2022-11-18 at 12 59 08 AM" src="https://user-images.githubusercontent.com/77593979/202640850-d801b825-1df1-4710-aced-661902fe8128.png">
<img width="545" alt="Screenshot 2022-11-18 at 12 59 34 AM" src="https://user-images.githubusercontent.com/77593979/202640926-42f0d50f-c2ec-4215-bc11-2bb288648f38.png">
